core/src/main/scala/org/apache/spark/sql/crossdata/execution/datasources/ddl.scala

```diff
@@ -220,10 +220,9 @@ private[crossdata] case class InsertIntoTable(tableIdentifier: TableIdentifier,
             if (sqlContext.catalog.tableHasIndex(tableIdentifier)) {
               //insert into ES because of global index
               val crossdataIndex = sqlContext.catalog.obtainTableIndex(tableIdentifier).get
+              val values = parsedRows map (x => x.mkString("\"", ",", "\"") ) mkString("(",",",")")
               sqlContext.sql(
-                s"""INSERT INTO ${crossdataIndex.indexIdentifier}
-                    |(${crossdataIndex.indexedCols.mkString(",")})
-                    |VALUES (${parsedRows.mkString(",")})""".stripMargin)
+                s"""INSERT INTO ${crossdataIndex.indexIdentifier.unquotedString} (${crossdataIndex.indexedCols.mkString(",")}) VALUES ${values}""".stripMargin)
```

Comment:
Is not better to use InsertIntoTable(tableIdentifier: TableIdentifier, parsedRows: Seq[DDLUtils.RowValues], schemaFromUser: Option[Seq[String]] = None).run(sqlContext)??